tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot

```diff
@@ -222,7 +222,9 @@ Spawn Notebook With Arguments  # robocop: disable
    ...              Environment variables can be passed in as kwargs by creating a dictionary beforehand
    ...              e.g. &{test-dict}  Create Dictionary  name=robot  password=secret
    [Arguments]  ${retries}=1  ${retries_delay}=0 seconds  ${image}=s2i-generic-data-science-notebook  ${size}=Small
-   ...    ${spawner_timeout}=600 seconds  ${gpus}=0  ${refresh}=${False}  ${same_tab}=${True}  &{envs}
+   ...    ${spawner_timeout}=600 seconds  ${gpus}=0  ${refresh}=${False}  ${same_tab}=${True}
+   ...    ${username}=${TEST_USER.USERNAME}  ${password}=${TEST_USER.PASSWORD}  ${auth_type}=${TEST_USER.AUTH_TYPE}
+   ...    &{envs}
```

Comment:
Are you sure this doesn't break existing calls to this kw?

Reply:
I haven't tested yet the regression, but there shouldn't be issue because I'm setting the default values. Will check

Reply:
my worry is that there might be calls like 
```
Spawn Notebook With Arguments    600s    1    False    True    <env-variables>
```

In which case <env-variables> might end up as `${username}` now

Reply:
An easy way to prevent this would be to put the new arguments after `&{envs}` (with default values), this should avoid any potential issues

Reply:
I tried so, but RF wants &{envs} as last argument because it detects it as kwargs